alws/crud/release.py

```diff
@@ -2,13 +2,25 @@
 
 from sqlalchemy.future import select
 from sqlalchemy.orm import Session, selectinload
+from sqlalchemy.sql.expression import func
 
 from alws import models
 
 
-async def get_releases(db: Session) -> typing.List[models.Release]:
-    release_result = await db.execute(select(models.Release).options(
+async def get_releases(page_number: typing.Optional[int],
+                       db: Session
+) -> typing.List[models.Release]:
+    query = select(models.Release).options(
         selectinload(models.Release.created_by),
         selectinload(models.Release.platform),
-    ).order_by(models.Release.id))
+    ).order_by(models.Release.id)
```

Comment:
Add `models.Release.id.desc()` instead `models.Release.id` so that releases are in the correct order